### PR TITLE
13.0 [MIG] base_tier_validation

### PR DESCRIPTION
**Pull request by spoerp**
Just edited one test which checked if there are any tier_validation records all ready created. Which failed when we installed purchase_tier_validation and then odoo.sh runs all the tests again.

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
Hi @spoerp , I'm very interested in having this module and base_tier_validation_formula running in v13. What's your migration status? Can I help?

File: base_tier_validation/models/tier_review.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    requested_by = fields.Many2one(
+        comodel_name="res.users",
+    )
+    reviewed_date = fields.Datetime(string='Validation Date')
+
+    @api.model
+    def _get_reviewer_fields(self):
+        return ['reviewer_id', 'reviewer_group_id', 'reviewer_group_id.users']
+
+    @api.multi
```
Comment (angelmoya) [inline base_tier_validation/models/tier_review.py:49]: delete api.multi

File: base_tier_validation/models/tier_validation.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    need_validation = fields.Boolean(compute="_compute_need_validation")
+    rejected = fields.Boolean(compute="_compute_validated_rejected")
+    reviewer_ids = fields.Many2many(
+        string="Reviewers", comodel_name="res.users",
+        compute="_compute_reviewer_ids",
+        search="_search_reviewer_ids",
+    )
+    can_review = fields.Boolean(compute="_compute_can_review")
+
+    @api.multi
```
Comment (angelmoya) [inline base_tier_validation/models/tier_validation.py:39]: delete

File: base_tier_validation/models/tier_validation.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    def _compute_can_review(self):
+        for rec in self:
+            rec.can_review = self.env.user in rec.reviewer_ids
+
+    @api.multi
```
Comment (angelmoya) [inline base_tier_validation/models/tier_validation.py:44]: delete

File: base_tier_validation/views/tier_definition_view.xml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="tier_definition_action" model="ir.actions.act_window">
+        <field name="name">Tier Definition</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">tier.definition</field>
+        <field name="view_type">form</field>
```
Comment (angelmoya) [inline base_tier_validation/views/tier_definition_view.xml:94]: delete view_type

**Comment by pegonzalezspesol** [comment]
@angelmoya @pedrobaeza I propose a new PR #111 because:
- @spoerp does not provide me feedback
- @spoerp did not sign the CLA
- I did test already the functionality together with purchase_tier_validation

**Comment by pedrobaeza** [comment]
Superseeded by #111 as no answer from original contributor.

**Pull request closed without merging**